src/stories/Library/Modals/modal-facet-browser/FacetBrowser.tsx

```diff
@@ -19,48 +19,46 @@ const FacetBrowser: React.FC<FacetBrowserProps> = ({
   showMore,
   showResults,
 }) => (
-  <div className="modal-backdrop">
-    <Modal shownModal={showModal} classNames="modal-right modal--no-padding">
-      <section className="facet-browser">
-        <header className="facet-browser__header">
-          <h3 className="text-header-h3">{title}</h3>
-          {clearAll && (
-            <button className="link-tag cursor-pointer facet-browser__clear-btn">
-              {clearAll}
-            </button>
-          )}
-        </header>
+  <Modal shownModal={showModal} classNames="modal-right modal--no-padding">
```

Comment:
Consider making `modal-right` a modifier i.e. `modal--right`.

It is outside the scope of this PR though.

Reply:
This would require some rewriting of the react components. I think this is a box I am not going to open.